src/js/components/tablero/tablero.scss

```diff
@@ -72,12 +74,26 @@
             cursor: pointer;
           }
         }
+        input {
```

Comment:
When you click on the input field within the filter, then the black border does not have rounded corners.

Reply:
Done